cdap-ui/server.js

```diff
@@ -112,4 +111,21 @@ cdapConfigurator.getCDAPConfig()
     });
 
     sockServer.installHandlers(server, { prefix: '/_sock' });
+    server.addListener('upgrade', function (req, socket, head) {
+      const nodejsserver = cdapConfig['dashboard.bind.address'];
+      const nodejsport = cdapConfig['dashboard.bind.port'];
+      const nodejsprotocol = (cdapConfig['ssl.external.enabled'] === 'true') ? 'https' : 'http';
+      let allowedOrigin;
+      if (nodejsport) {
+        allowedOrigin = `${nodejsprotocol}://${nodejsserver}:${nodejsport}`;
```

Comment:
`127.0.0.1` and `localhost` should be interchangeable. Currently cdap-default for sandbox is bind to `127.0.0.1`